ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -350,17 +384,21 @@ runDataDiffusion tracers
         (
           case address of
                Left sd -> return sd
-               Right a -> Snocket.open sn (Snocket.addrFamily sn a)
+               Right addr -> do
+                 traceWith dtDiffusionInitializationTracer $ CreatingServerSocket addr
```

Comment:
The same note as for the other `Creating...` message applies, e.g. if things go wrong the diffusion will error.